core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -2149,7 +2149,11 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
 #if !defined(OS_ENABLE_HW_BOUND_CHECK)              \
     || WASM_CPU_SUPPORTS_UNALIGNED_ADDR_ACCESS == 0 \
     || WASM_ENABLE_BULK_MEMORY != 0
+#if WASM_ENABLE_THREAD_MGR == 0
```

Comment:
You may want to be consistent with what we do at the beginning of the function: https://github.com/bytecodealliance/wasm-micro-runtime/blob/4aee3cf14e471eff3f0a1e731423ad35243b9991/core/iwasm/interpreter/wasm_interp_classic.c#L1160-L1164

Also, similar changes need to be made for `wasm_interp_fast.c`. See https://github.com/bytecodealliance/wasm-micro-runtime/pull/2834

Reply:
Thanks for the review. I just pushed another commit that makes the check consistent and applies the same fix to `wasm_interp_fast.c`.